lib/src/rewrite.rs

```diff
@@ -467,18 +467,21 @@ pub struct MoveCommitsStats {
 /// Moves `target_commits` from their current location to a new location in the
 /// graph.
 ///
-/// The roots of `target_commits` are rebased onto the new parents
+/// Commits in `target_roots` are rebased onto the new parents
 /// given by `new_parent_ids`, while the `new_children` commits are
-/// rebased onto the heads of `target_commits`. This assumes that
-/// `target_commits` and `new_children` can be rewritten, and there
-/// will be no cycles in the resulting graph. `target_commits` should
-/// be in reverse topological order.
+/// rebased onto the heads of `target_commits`. If `target_roots` is
+/// empty, it will be computed as the roots of the connected set of
```

Comment:
nit: empty -> `None`

Do we have other expectations to document (such as `target_roots` should be a subset of `target_commits`)?